code/datums/martial.dm

```diff
@@ -66,12 +66,11 @@
 		H.verbs += help_verb
 	if(make_temporary)
 		temporary = 1
-	if(H.martial_art && H.martial_art.temporary)
-		if(temporary)
+	if(temporary)
+		if(H.martial_art)
```

Comment:
That isn't actually needed. Even if `H.martial_art` isn't permanent, its `base` will be the permanent art and that's what's used under this if statement.